packages/react-composites/tests/app/call/QueryArgs.ts

```diff
@@ -29,6 +29,7 @@ export interface QueryArgs {
   groupId: string;
   token: string;
   displayName: string;
+  makeMeLotsOfPeople: boolean;
```

Comment:
while I do like the name, its a little unclear on the purpose - are you making me (the test user) into lots of people? or are you making lots of people for the test user .. as other remote participants?

```suggestion
  mockRemoteParticipantCount: number
```